src/main/resources/db/migration/0250/V279__SubmissionSnapshots.sql

```diff
@@ -0,0 +1,5 @@
+CREATE TABLE accelerator.submission_snapshots (
+id BIGINT PRIMARY KEY GENERATED BY DEFAULT AS IDENTITY,
+    file_id BIGINT REFERENCES files ON DELETE SET NULL,
```

Comment:
Why `ON DELETE SET NULL` instead of `ON DELETE CASCADE` here? What does it mean for a row in this table to have a null file ID?